pandas/tests/extension/base/reshaping.py

```diff
@@ -249,6 +249,9 @@ def test_merge_on_extension_array_duplicates(self, data):
         )
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.filterwarnings(
+        "ignore:The previous implementation of stack is deprecated"
```

Comment:
Just for my edification, when this deprecation is enforced will we need to change all/most of these unit tests?

Reply:
Correct - but all tests that use stack test with both `future_stack=False` and `future_stack=True`. So it will just be ripping out the `future_stack=False` case.